core/templates/base-components/oppia-root.directive.ts

```diff
@@ -22,7 +22,7 @@ import { OppiaAngularRootComponent } from
   'components/oppia-angular-root.component';
 
 angular.module('oppia').directive('oppiaRoot', [
-  '$translate', function($translate) {
+  '$rootScope', '$translate', function($rootScope, $translate) {
```

Comment:
@srijanreddy98, Can we remove all the occurrence `$rootScope.$applyAsync()` and resolve #8521?  [This will help us avoid further addition of `$rootScope.$applyAsync()` in the codebase]